noetic/distribution.yaml

```diff
@@ -5148,7 +5148,7 @@ repositories:
       tags:
         release: release/noetic/{package}/{version}
       url: https://github.com/NOKOV-MOCAP/mocap_nokov_release.git
-      version: 0.0.1-1
+      version: 0.0.1-2
```

Comment:
I see the [latest tag is `0.0.2`](https://github.com/NOKOV-MOCAP/mocap_nokov/releases/tag/0.0.2), but this says to re-release `0.0.1` which seems incorrect. I also see `0.0.2` and `0.0.1` point to the same commit `e332cff09f3ed781295ff17d12a60494668cf68b`. It seems like something went wrong with the release process. Did `0.0.1` always point to that commit?